conf/defaults.ini

```diff
@@ -288,6 +288,11 @@ application_insights_endpoint_url =
 # Controls if the UI contains any links to user feedback forms
 feedback_links_enabled = true
 
+# Defines external auth providers (such as OAuth) whos ID is used to identify to analytics services
+[analytics.external_id_auth_modules]
+oauth_grafana_com = true
+oauth_grafananet = true
```

Comment:
One disadvantage with this structure is that it can't be set using environmental variables for the same reasons as provided in https://github.com/grafana/grafana/issues/67230

Reply:
Could make it a single key of comma seperated values, like feature toggles also?

I don't have a preference, I'm happy to do whatever we prefer here, but I don't really want to do it _both_ ways like feature toggles.

Reply:
Yeah, I'm with you on that. I _think_ I'd weakly prefer using `util.SplitString` to make this a list that can be provided through envvars. Though separate lines has it's advantages in that it looks nice 🤔